clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/EcsSpec.java

```diff
@@ -62,7 +61,7 @@
 
   @LocalServerPort private int port;
 
-  @MockBean AmazonClientProvider mockAwsProvider;
+  @MockBean protected AmazonClientProvider mockAwsProvider;
```

Comment:
Curios, why are we reducing the scope here? Would it make sense to also reduce the scope of all public mocked beans?

Reply:
AFAIK the default scope is package-private, so this is _increasing_ scope to allow the tests in `CreateServerGroupSpec` to setup mocked behavior against this bean. Maybe it would be better to explicitly set the other bean in this package to `private` to make this distinction more explicit?

Reply:
Ah that's fair (I just assumed that anything that extended this was in the same package, but it's not necessarily true).